web/models/notes.ex

```diff
@@ -7,4 +7,10 @@ defmodule Notox.Note do
 
     timestamps
   end
+
+  def changeset(model, params \\ %{}) do
+    model
+    |> cast(params, [:title ,:description])
```

Comment:
I'd probably do something like:


```elixir
@required_fields [
  :title,
  :description,
]
@optional_fields @required_fields
```

Since the optional fields will always be a superset of the required fields, you can just tack on extra fields later if they diverge like:

```elixir
@optional_fields @required_fields ++ [
  :some_other_field
]
```